### ISSUE TITLE
Future of this project - part 2

### ISSUE THREAD

**Issue by Qu4tro** [jpadilla/django-rest-framework-yaml#21]
I've forked it. 

`pip install drf-yaml` at [Qu4tro/drf-yaml](https://github.com/Qu4tro/drf-yaml)

@jpadilla Let me know if you want to do anything. i.e Integrate the fork changes into the original or something else. I'm fine with the status quo.

**Comment by stefan6419846** [comment]
I would like to see the changes incorporated into upstream as well, but for now all contact attempts seem to have failed. My last attempt has been about three months ago via e-mail, but without any response.

**Comment by Qu4tro** [comment]
Yeah, I stalked it for a bit to see if it would pop again, but no dice. Picked up a project where I needed something like this and decided to just go ahead and fork it.

I've modernized the tooling and fixed a couple of things. I think the only issue / PR that's still relevant in the new repo is #16. I have ideas for it and I'll need it, so I'll implement it soon, but it wasn't on my priority list.

For what it's worth, I'm open for the new repo to be merged upstream at any point in the future.

**Comment by jpadilla** [comment]
@Qu4tro I'm open to adding you as a maintainer to this repo / package, if you'd like.

**Comment by Qu4tro** [comment]
Hey. 

Yes, that would be fine.

I'll prep. the PR for upstream this weekend then.

**Comment by Qu4tro** [comment]
@jpadilla what's the best way to contact you 1-on-1? 

I will need to set some environment variables on the CI, as that's a blocker for a new PyPi release. I would like to discuss a few quick things, like the compat.py (which I had dropped).

**Comment by stefan6419846** [comment]
Has there been any progress on this in the meantime? As far as I can see, the latest release still only refers to ancient Python and Django versions.
